app/views/lecturers/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Editing Lecturers</h1>
```

Comment:
We dont need views for lecturers. the info for lecturers will be presented with the associated resource, subjects